DevOps/README.md

```diff
@@ -19,6 +19,7 @@
 - [Linux Command Cheatsheet](https://www.guru99.com/linux-commands-cheat-sheet.html)
 - [Linux Masterclass Course - Apoorv Goyal(YouTube)](https://www.youtube.com/playlist?list=PL2kSRH_DmWVZp_cu6MMPWkgYh7GZVFS6i)
 - [Bogdan Stashchuk - Linux for beginners playlist](https://www.youtube.com/playlist?list=PLWkguCWKqN9OrwsklvLC8FB87TbaOQGqY)
+- [Linux Kernal Documentation](https://docs.kernel.org/)
```

Comment:
I think this section to help people understand about kernel to a level that they can get started with DevOps, Kernel documentation might not be a great resource for this purpose so I feel this should be removed